sdks/java/io/debezium/src/main/java/org/apache/beam/io/debezium/Connectors.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.io.debezium;
+
+import io.debezium.connector.db2.Db2Connector;
+import io.debezium.connector.mysql.MySqlConnector;
+import io.debezium.connector.oracle.OracleConnector;
+import io.debezium.connector.postgresql.PostgresConnector;
+import io.debezium.connector.sqlserver.SqlServerConnector;
+import org.apache.kafka.connect.source.SourceConnector;
+
+/** Enumeration of debezium connectors. */
+public enum Connectors {
+  MYSQL("MySQL", MySqlConnector.class),
+  POSTGRES("PostgreSQL", PostgresConnector.class),
+  SQLSERVER("SQLServer", SqlServerConnector.class),
+  ORACLE("Oracle", OracleConnector.class),
+  DB2("DB2", Db2Connector.class),
```

Comment:
If the dependencies are optional, then we may also need to make these classes into strings, so that they will not require the class being in the classpath.

Reply:
Done